images/tidb-operator/Dockerfile

```diff
@@ -1,8 +1,15 @@
 FROM alpine:3.14
 
 ARG TARGETARCH
+ARG RCLONE_VERSION=v1.51.0
 RUN apk add tzdata bind-tools --no-cache
 ADD bin/${TARGETARCH}/tidb-scheduler /usr/local/bin/tidb-scheduler
 ADD bin/${TARGETARCH}/tidb-discovery /usr/local/bin/tidb-discovery
 ADD bin/${TARGETARCH}/tidb-controller-manager /usr/local/bin/tidb-controller-manager
 ADD bin/${TARGETARCH}/tidb-admission-webhook /usr/local/bin/tidb-admission-webhook
+
+RUN wget -nv https://github.com/ncw/rclone/releases/download/${RCLONE_VERSION}/rclone-${RCLONE_VERSION}-linux-${TARGETARCH}.zip \
```

Comment:
We can use aws-go-sdk to avoid adding a new binary to tidb-operator's image. There is a simple example in https://github.com/aws/aws-sdk-go#complete-sdk-example.

Reply:
there are 3reasons here to use rclone:
1. general handling for remote storage, aws, gcp, ceph, aruze etc. share the same interface rclone copy ....
2. currently, tide-backup-manager uses the rclone, it may be easy to maintain
3. we do not need the complex requirement, only need copy/upload/download. rclone may better choice.

Reply:
fixed in https://github.com/pingcap/tidb-operator/pull/4784/commits/75fa349f322cb6c9de5b3a7d7ef7655b35ff4d3f